sns.tf

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (C) 2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_sns_topic" "shuntingyard_ops_sns" {
```

Comment:
What will be the purpose of this SNS topic ?

Reply:
sending cloudwatch alerts. We can in the client code assign it to send email or messages via slack for instance.